src/file_io.jl

```diff
@@ -1008,6 +1041,10 @@ end
                                   moments.charged.qpar.data, t_idx, z, r, n_ion_species)
             append_to_dynamic_var(io_moments.thermal_speed, moments.charged.vth.data,
                                   t_idx, z, r, n_ion_species)
+            append_to_dynamic_var(io_moments.chodura_integral_lower, moments.charged.chodura_integral_lower.data,
+                                  t_idx, r, n_ion_species)
+            append_to_dynamic_var(io_moments.chodura_integral_upper, moments.charged.chodura_integral_upper.data,
+                                  t_idx, r, n_ion_species)
```

Comment:
Here need to copy the fixes from the above non-debug version (once we know that they work!). Otherwise the I/O will break when we debug shared-memory.

This is making me think there must be a nicer way to handle the non-debug vs. debug to avoid code duplication, but that's for another day!